src/components/directory-breadcrumbs.js

```diff
@@ -76,18 +98,17 @@ const DirectoryBreadcrumbs = () => {
         ) {
             return (
                 <Typography className={classes.directory} color="textPrimary">
-                    <FolderOpenIcon className={classes.icon} />
+                    {currentPath.length === 1 ? (
+                        <FolderOpenIcon className={classes.icon} />
+                    ) : null}
                     {currentPath[currentPath.length - 1].elementName}
                 </Typography>
             );
         }
     };
 
     return (
-        <Breadcrumbs maxItems={10} aria-label="breadcrumb">
-            {selectedDirectory !== null && (
-                <ArrowForwardIosIcon className={classes.icon} />
-            )}
+        <Breadcrumbs aria-label="breadcrumb" className={classes.breadcrumbs}>
```

Comment:
why you removed maxItems={10}

Reply:
Because we don't want the collapsed breadcrumbs functionality for now.